io.catenax.sealant.sealant_pass/1.0.0/SealantPass.ttl

```diff
@@ -0,0 +1,648 @@
+    bamm:description "Entity encapsulating the regulatory information. "@en.
+:limitations a bamm:Property;
+    bamm:characteristic bamm-c:Text;
+    bamm:preferredName "limitations"@en;
+    bamm:description "Information shall be provided regarding relevant Union safety, health and environmental provisions including the prohibitions and restrictions"@en;
+    bamm:exampleValue "Regulation (EC) No 1005/2009".
+:authorSDS a bamm:Property;
+    bamm:characteristic bamm-c:Text;
+    bamm:preferredName "author SDS"@en;
+    bamm:description "SDS Author changing the document. "@en;
+    bamm:exampleValue "Max Mustermann".
+:otherInformation a bamm:Property;
+    bamm:characteristic :OtherInformationCharacteristic;
+    bamm:preferredName "other information"@en;
+    bamm:description "Additional information relevant to the SDS document. "@en.
+:OtherInformationCharacteristic a bamm-c:Set;
+    bamm:dataType :OtherInformationEntity;
+    bamm:preferredName "other information characteristic"@en;
+    bamm:description "Describes additional information, in a set of values. "@en.
+:OtherInformationEntity a bamm:Entity;
+    bamm:properties (:indicationChanges :authorSDS);
+    bamm:preferredName "other information entity"@en;
+    bamm:description "Entity encapsulating the required information."@en.
+:indicationChanges a bamm:Property;
+    bamm:characteristic bamm-c:Text;
+    bamm:preferredName "indication changes"@en;
+    bamm:description "in the case of a revised safety data sheet, a clear indication of where changes have been made to the previous version of the safety data sheet."@en;
+    bamm:exampleValue "Created 1.1.2000, Updated 24.3.2019".
+:exposureLimit a bamm:Property;
+    bamm:characteristic :ListOfIngredients;
+    bamm:preferredName "exposure limits"@en;
+    bamm:description "Information about the applicable exposure limits of the substances and mixtures. (SDS Section 8.1)"@en.
+:supplierName a bamm:Property;
+    bamm:characteristic bamm-c:Text;
+    bamm:preferredName "supplier name"@en;
+    bamm:description "Name of the Supplier. "@en;
+    bamm:exampleValue "Henkel".
+:IngredientsEntity a bamm:Entity;
+    bamm:properties (:chemicalName :chemicalPercentage);
+    bamm:preferredName "ingredients entity"@en;
+    bamm:description "Entity encapsulating the ingredients. "@en.
+:chemicalName a bamm:Property;
+    bamm:characteristic bamm-c:Text;
+    bamm:preferredName "chemical name"@en;
+    bamm:description "Name of the chemical. "@en;
+    bamm:exampleValue "Chemical A1".
+:chemicalPercentage a bamm:Property;
+    bamm:characteristic :ChemicalPercentageTrait;
+    bamm:preferredName "chemical percentage"@en;
+    bamm:description "Describes chemical distribution in percentage. It can include a percentage range. Either 10% or 5,5%-6% . Allowed are decimal values with a \",\" seperator. Mandatory are no spaces inbetween the values and the percentage character. "@en;
+    bamm:exampleValue "32,3 %"^^xsd:decimal.
```

Comment:
this is not a valid decimal value. Should be 32.3 (due to that the model does not validate)